data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "Giving Green",
+            link: "https://www.idinsight.org/givinggreen",
+            keywords: ["ngo", "efficiency-assessment"],
+            label: "giving-green",
+        },
+        description: {
+            content: "[IDinsight](https://www.idinsight.org/) works on assessing the efficiency (ratio of effort to impact) of initiatives fighting poverty."
+            + " Giving Green is a project by IDinsight that tries to give clear funding recommendations."
+            + " The website describes the project as being in Beta stage. Many of the recommendations are drawn from [IDinsight itself](https://www.idinsight.org/giving-green-short-term-recommendations)"
+            + " or are sourced from a [report by the Founder's Pledge](https://founderspledge.com/stories/climate-change-executive-summary) and [recommend initiatives focusing on US politics or the REDD+ mechanism](https://www.idinsight.org/giving-green-long-term-recommendations)."
+            + " Giving Green attempts to solve a key question in the fight against the climate crisis but seems to be in a very early stage in doing so.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "Clean Air Taskforce",
+            link: "https://www.catf.us/",
+            keywords: ["ngo", "legislation", "energy", "technology"],
+            label: "catf",
+        },
+        description: {
+            content: "The CATF lobbies US governmental institutions to reduce emissions of greenhouse gases."
+            + " In addition they work to structure future [technical options for carbon-reduced energy production](https://www.catf.us/work/advanced-energy-systems/)."
+            + " CATF aims to assist corporates and governments in designing future energy systems based on renewables, biofuels, nuclear and systemic improvements of grids and consumption.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "ImpactMatters - Climate Change",
+            link: "https://www.impactmatters.org/top-lists/climate-change/",
+            keywords: ["ngo", "impact-assessment", "tree-planting"],
+            label: "impact-matters",
+        },
+        description: {
+            content: "Here, Impact Matters analyses the efficiency of tree planting initiatives."
+            + " It is an attempt to provide donors with the most efficient ways of fighting climate change."
+            + " While tree planting is important, changing public opinion and governments is arguably more impactful."
+            + " Also they seem to rely on impact statements by the organizations they list. There seems to be no independent scientific assessment." 
+            + " That makes initiatives overstating their impact more likely to receive funding.",
```

Comment:
delet. Opinion

Reply:
In my understanding it's not an opinion but a logical conclusion that if impact is not comparable, the likelihood that funds are spent inefficiently is greater than zero, i.e. it's fair to assume that funds are not put to the best use.

Reply:
Also their statement `We analyzed tree planting nonprofits and identified three that are highly cost-effective` is at least highly misleading considering they only scrolled through webpages. We're not pretending to have "analyzed" any initiative either. And if we do, we provide that analysis

Reply:
I added another paragraph: `Their claim to have analyzed the suggested initiatives can be considered misleading. The same holds for their suggestion to offset years of your personal carbon emissions without asking anything about your consumption`

Reply:
I like that!